source/16-6-2_Outcomes.markdown

```diff
@@ -70,15 +104,73 @@ If you have more than one measure to add to a category, you can also opt to put
 - Click on 'bulk add new {{outcome}} measures.
 - Enter the measures in the text box, each one separated by a semi-colon. If you choose to add them in this way, you'll need to find each one in the list and edit it (click on the pen and paper icon to the right to open the editing box) to include the type, values and notes, as we saw with the single measure before. 
 
-[Editing Icon](16.6.2f.png)
+![Editing Icon](16.6.2f.png)
 
 - Continue these steps until you have added all the measures that you need.
 
+#### Scores with user specified issue
+
+This measure type is intended to be used where you want to record a score, but also want to record what the issue is.
+One option is to create a set of different measures, one for each type of issue, and recording the detail in the 
+comments box. However this can feel prescriptive and may not be appropriate for your situation. It can also lead to 
+a long list that becomes hard to navigate.
+
+The "scores with user specified issue" measure type allows you to record the score and the issue in one place. The 
```

Comment:
The **"**scores with user specified issue" measure type allow**s** you to record
The scores with **a "**user specified issue" measure type**,** allow you to record
**ADJUSTMENTS**

Reply:
I think this is the label in the system so I've not made this change, but instead:

The measure type "scores with user specified issue" allows you to